xml_converter/proto/CMakeLists.txt

```diff
@@ -0,0 +1,11 @@
+INCLUDE(FindProtobuf) 
+FIND_PACKAGE(Protobuf REQUIRED)
```

Comment:
Do you need `FIND_PACKAGE(Protobuf REQUIRED)` both here and in the main CMake file? Or can one of them be eliminated?

Reply:
I did tests and the FIND_PACKAGE line can be removed. INCLUDE is required.